cmd/sniffer/main.go

```diff
@@ -27,9 +26,9 @@ const (
 
 var (
 	iface      = flag.String("i", "eth0", "Interface to get packets from")
-	dstport    = flag.Uint("p", 9092, "Kafka broker port") // todo: use -f tcp and dst port 9092
+	dstport    = flag.Uint("p", 9092, "Kafka broker port")
 	snaplen    = flag.Int("s", 16<<10, "SnapLen for pcap packet capture")
-	filter     = flag.String("f", "tcp", "BPF filter for pcap")
+	filter     = flag.String("f", fmt.Sprintf("tcp and dst port %d", *dstport), "BPF filter for pcap")
```

Comment:
Yes, it worked for me - got no packets skipped due to check of the port filter in the loop. And no packets were lost

Reply:
Okay, let's remove `-f` flag then:
```
filter     = fmt.Sprintf("tcp and dst port %d", *dstport)
```

Reply:
> got no packets skipped

it didn't work because some excess packets were passed, not because something was skipped (that's why I've added this check: `if tcp.DstPort != layers.TCPPort(*dstport) {`)